spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/aggregation/DateOperators.java

```diff
@@ -192,10 +192,10 @@ public static Timezone ofExpression(AggregationExpression expression) {
 	 */
 	public static class DateOperatorFactory {
 
-		private final @Nullable String fieldReference;
-		private final @Nullable Object dateValue;
-		private final @Nullable AggregationExpression expression;
-		private final Timezone timezone;
+		protected final @Nullable String fieldReference;
```

Comment:
Let's keep these `private`, otherwise implementation details leak into subclasses. You can still keep these fields in your subclass.